### ISSUE TITLE
Determine how to use CEDAR metadata

### ISSUE THREAD

**Issue by ryscher** [CDL-Dryad/dryad-product-roadmap#2049]
Once we have collected metadata through CEDAR, we should make it accessible. The first step is defining how it should be accessible:
- Who will use this metadata? How?
- Should the JSON file be downloadable? Should it be included in the zip file with the rest of the dataset? Available through the API?
- Should it be rendered on the landing page?
- How can it be used in the search system?

**Comment by ryscher** [comment]
According to the BRIDGE grant, we will make it accessible on the landing page.

**Comment by ryscher** [comment]
After discussion with Ted, this has been put on hold. We need to find a user community for CEDAR metadata before we can decide how to make use of the metadata.

**Comment by ahamelers** [comment]
Is this closed in #2866 ?

**Comment by ryscher** [comment]
Yes, we can consider it closed.
